test/util/testnode/config.go

```diff
@@ -99,6 +99,12 @@ func (c *Config) WithGensisTime(t time.Time) *Config {
 	return c
 }
 
+// WithCommitTimeout sets the CommitTimeout and returns the Config.
+func (c *Config) WithCommitTimeout(d time.Duration) *Config {
```

Comment:
should we use TimeoutCommit instead of CommitTimeout here just for consistency?

Reply:
Of course, here is the fix https://github.com/celestiaorg/celestia-app/pull/2220/commits/4a29d4fc2385a517869757250116f890dc4c53d2.